### PR TITLE
#2383 Allow admins to delete scores

### PR DESCRIPTION
**Pull request by shaunxp20**
This will give admins the ability to delete submitted scores. Some notes:

 * It re-uses the existing delete button/functionality for deleting suspicious scores (I moved these buttons into a partial)
 * I refactored/renamed `awaiting_review(account, &block)` to `pending_approval?`

### CODE REVIEW

File: spec/models/submission_score_spec.rb [hunk 1]
```diff
@@ -244,6 +244,78 @@
...
+          end
+
+          it "updates the team's total number of completed scores" do
+            expect(team_submission.
+              public_send("complete_#{judging_round}_submission_scores_count")).to eq(2)
+          end
+
+          it "updates the team's total number of offical completed scores" do
+            expect(team_submission.
+              public_send("complete_#{judging_round}_submission_scores_count")).to eq(2)
```
Comment (stenington) [inline spec/models/submission_score_spec.rb]: This is checking the same thing as the previous test, I think you wanted `complete_#{judging_round}_official_submission_scores_count` right?

File: app/assets/stylesheets/_forms.scss [hunk 2]
```diff
@@ -288,4 +288,8 @@ form[data-remote] {
   display: flex;
   justify-content: space-between;
   margin: 1rem 0;
-}
\ No newline at end of file
+
+  .delete-score {
```
Comment (stenington) [inline app/assets/stylesheets/_forms.scss]: I prefer utility classes for this kind of thing, something like `.margin--l-auto` could be added [here](https://github.com/Iridescent-CM/technovation-app/blob/qa/app/assets/stylesheets/_utilities.scss#L99), although I'm willing to do it this way too. I think utility classes are easier to reuse for this kind of tweak though. Your call! 😄

File: spec/models/submission_score_spec.rb (hunk 1 above)
Comment (shaunxp20) [reply spec/models/submission_score_spec.rb]: Yup! Great catch, thanks @stenington! 🦅 👁 I updated this! 👍

File: app/assets/stylesheets/_forms.scss (hunk 2 above)
Comment (shaunxp20) [reply app/assets/stylesheets/_forms.scss]: Thanks for the feedback, I was actually going to ask about this! I made it a utility class! 👍 What's funny though is that very recently I was looking at Tailwind and Tachyons; a couple of weeks ago I wouldn't have known the term utility class. 😄 😅 😀

Comment (stenington) [reply app/assets/stylesheets/_forms.scss]: I really just know it from Bootstrap, which has a pretty complete set of spacing utility classes: https://getbootstrap.com/docs/4.4/utilities/spacing/

**Comment by stenington** [comment]
Changes look good 👍

**Pull request merged**